js/components/accessibility/autofocus.jsx

```diff
@@ -0,0 +1,55 @@
+/**
```

Comment:
This is not .jsx file since it doesn't contain any React template. Create folder `utils` please and put this file there. This is just .js.
Also since it's a brand new file I'd suggest to not rely on the controller at all. Make these methods more atomic. `getToggleButtons` looks private, not public. `configureMenyAutofocus` should expect not `controller` but more precisely two params: `menu`, `controller.state`, `controller.toggleButtons`.

Reply:
@zelibobla 
> This is not .jsx file since it doesn't contain any React template. This is just .js.
Yes. I did it automaticaly! Thank you!

> Create folder utils please and put this file there. 
\+

> getToggleButtons looks private, not public
it is public method. It is usable in controlBar

> configureMenyAutofocus should expect not controller but more precisely two params
\+